### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by coderschoolreview** [tam01235/MyTODO#1]
### Good stuffs
- Nice use `RecyclerView` and `CustomAdapter` instead of `ListView`.
- Good to see you use `SQLite` to store your data.
- Nice use `Splash Screen` and Animation for many `View`.
- UI has been improved significantly from the minimum.
### Suggestions
- Many things need to talk about your code because it isn't uniquely well-organized and formatted. Try to organize them into sub-package and follow `Java Naming Convention`.
- I suggest not use hard-code in any cases even in layout file. It's super hard to maintain later.
- Remember to declare `Access Modifier` for all variables.
- Close `Cursor` before using in `SQLite`.

Overall your submission is good.Hope to see you keep moving up

**Comment by nhoxbypass** [comment]
Thank you very much for reviewing my code, it's quite a mess!
 I'll try to organize and format my code base on Design Pattern such as MVP or MVVM and try to naming variables more correctly.

**Comment by nongdenchet** [comment]
Hi I want to ask some question:
1. How well do you know MVP and MVVM?
2. Do you familiar with RxJava, RxAndroid or Dependency Injection?

If the answers are yes, I think you should not waste time taking this course

**Comment by nhoxbypass** [comment]
No. I've just heard MVVM and MVP in the training of FPT software and GDG Vietnam 3 months ago, but im still don't really understand and it's hard for me to implement it.
With RxJava I'm just starting to learn too.
